### PR TITLE
Turn exercise 004 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**
Fix PR #2168

### CODE REVIEW

**Comment by Ozyugoo** [comment]
@cuihtlauac I think I have resolved it as directed, kindly check it out.

File: src/ocamlorg_frontend/css/other/syntax.css
Comment (cuihtlauac) [inline src/ocamlorg_frontend/css/other/syntax.css:1]: Remove this file from this PR

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: src/ocamlorg_frontend/css/other/syntax.css
Comment (cuihtlauac) [inline src/ocamlorg_frontend/css/other/syntax.css:1]: You need to delete the file from the PR, not from the project.

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: src/ocamlorg_frontend/css/other/syntax.css
Comment (Ozyugoo) [reply src/ocamlorg_frontend/css/other/syntax.css:1]: I don't really understand how to go about it, could you please point me in the right direction?

Comment (Ozyugoo) [reply src/ocamlorg_frontend/css/other/syntax.css:1]: I think I understand now

**Pull request closed without merging**